### PR TITLE
concurrent batch queries

### PR DESCRIPTION
**Pull request by VladVega**
@davidvgalbraith here is what I would need to do in order to have batch query concurrency. Worth it?

### CODE REVIEW

File: lib/read.js [hunk 1]
```diff
@@ -258,12 +258,22 @@ class ReadSql extends AdapterRead {
...
                 });
-                return this.formatPoints(points);
+                return points;
             });
         }, { concurrency: BATCHING_CONCURRENCY })
         .then((res) => {
+            var points = _.chain(res)
+            .filter('length')
+            .sortBy(function(pts) {
+                return JSON.stringify(pts[0][timeField]);
```
Comment (davidvgalbraith) [inline lib/read.js]: Do you need to JSON.stringify? The points don't have JuttleMoments yet I think.

Comment (VladVega) [inline lib/read.js]: They are actually: https://github.com/juttle/juttle-sql-adapter-common/pull/47/files#diff-77960d7f37d725c98368151aa4bc18f7L255

Comment (davidvgalbraith) [inline lib/read.js]: Ok, can you `flatten` before you `sortBy` and get rid of the `filter` and `[0]`?

**Comment by VladVega** [comment]
@davidvgalbraith ok check it out - new logic for null aggregates.

**Comment by davidvgalbraith** [comment]
+1

**Pull request merged**